projects/poppler.freedesktop.org/package.yml

```diff
@@ -0,0 +1,63 @@
+distributable:
+  url: https://poppler.freedesktop.org/poppler-{{version}}.tar.xz
+  strip-components: 1
+
+versions:
+  github: freedesktop/poppler/tags
+  strip: /^poppler-/
+
+provides:
+  - bin/pdfattach
+  - bin/pdfdetach
+  - bin/pdffonts
+  - bin/pdfimages
+  - bin/pdfinfo
+  - bin/pdfseparate
+  - bin/pdftocairo
+  - bin/pdftohtml
+  - bin/pdftoppm
+  - bin/pdftops
+  - bin/pdftotext
+  - bin/pdfunite
+
+# TODO: Add in libopenjpeg, libtiff
+dependencies:
+  boost.org: '>=1.58.0'
+  cairographics.org: '>=1.16.0'
+  curl.se: '*'
+  freedesktop.org/fontconfig: '>=2.13'
+  freetype.org: '>=2.10'
+  gnome.org/glib: '>=2.64'
+  libjpeg-turbo.org: '*'
+  libpng.org: '*'
+  openjpeg.org: '*'
+  poppler.freedesktop.org/poppler-data: '*'
```

Comment:
I would have preferred `poppler.freedesktop.org/data` but the beauty of namespacing is this is contained so not important.